trinity/nodes/full.py

```diff
@@ -35,10 +35,10 @@ def get_p2p_server(self) -> BaseService:
             self._p2p_server = Server(
                 self._node_key,
                 self._node_port,
-                manager.get_chain(),
-                manager.get_chaindb(),
+                manager.get_chain(),  # type: ignore
```

Comment:
Doesn't seem like we have a proper class for that. Was unsure how to resolve that and ignored for now.